map/lanelet2_extension/package.xml

```diff
@@ -24,6 +24,7 @@
   <depend>geometry_msgs</depend>
   <depend>visualization_msgs</depend>
   <depend>pugixml-dev</depend>
+  <depend>eigen</depend>
```

Comment:
Same here. No need of adding eigen to lanelet2_extension package.

Reply:
@mitsudome-r this is fixed